### PR TITLE
LPS-122103 Create test to check if calculator expression have the field labels and if its field options show the field references

### PR DESCRIPTION
**Pull request by alinedoleron**

### CODE REVIEW

**Comment by alinedoleron** [comment]
ci:test:sf

**Comment by alinedoleron** [comment]
ci:test:relevant

**Comment by alinedoleron** [comment]
ci:test:relevant

**Comment by natocesarrego** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/components/RuleBuilder/RulesSupport.es.js [hunk 1]
```diff
@@ -322,6 +322,22 @@ const findInvalidRule = (rule) => {
 	return findRuleByFieldName('', [rule]);
 };
 
+const replaceFieldIdByFieldLabel = (expression, fields) => {
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/components/RuleBuilder/RulesSupport.es.js:325]: @alinedoleron, shouldn't we name it `replaceFieldNameByFieldLabel` instead?

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/test/js/components/RuleList/RuleList.es.js [hunk 2]
```diff
@@ -158,6 +191,34 @@ const configDefault = {
 	spritemap,
 };
 
+const configNumeric = {
+	pages: pagesWithNumbers,
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/test/js/components/RuleList/RuleList.es.js:195]: @alinedoleron, can you please declare the `pagesWithNumbers` just before the `configNumeric`?

From what I could see from this code, the variables are being declared using the alphabetic order except for the variables that are dependencies of other variables, which is the case of variables `pages` and `pagesWithNumbers`.

**Review by natocesarrego — CHANGES_REQUESTED** [review]
Good pull, @alinedoleron! 😄 

See below my two comments.

Thanks.

**Comment by alinedoleron** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**